.cargo/config.toml

```diff
@@ -0,0 +1,5 @@
+[env]
+# Required for rust toolchains 1.70.0+ until this commit
+# https://github.com/paritytech/substrate/commit/dd45632f298db02f6946c906296bbf86a74f4f46
+# is included in a release and we update to depend on it for the substrate-wasm-builder
+WASM_BUILD_RUSTFLAGS = "-C target-cpu=mvp -C target-feature=-sign-ext"
```

Comment:
I'm not sure about it becoming obsolete because the polkadot-v0.9.41 branch doesn't have the flags included yet either: https://github.com/paritytech/substrate/blob/polkadot-v0.9.41/utils/wasm-builder/src/wasm_project.rs#L635

And the RUSTC_VERSION for local dev environments isn't mentioned anywhere in the README.md's, but more importantly, if we want to use a newer `rustc` it won't work without these flags.

In defense of the flags themselves:
- Don't think we need the sign-extended feature atm, especially if it breaks things
https://github.com/WebAssembly/spec/blob/main/proposals/sign-extension-ops/Overview.md
- Parity Tech added the mvp cpu target in other crates' wasm builds also
https://github.com/paritytech/cargo-contract/pull/838